components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -133,27 +127,16 @@ abstract class AbstractFetchDownloadService : Service() {
     internal fun setDownloadJobStatus(downloadJobState: DownloadJobState, status: DownloadJobStatus) {
         synchronized(context) {
             if (status == DownloadJobStatus.ACTIVE) { downloadJobState.notifiedStopped = false }
-            downloadJobState.status = status
+            downloadJobState.state.status = status
```

Comment:
You also have to update the store, you could use  [updateDownloadState](https://github.com/mozilla-mobile/android-components/blob/cdef87e847f4272fcb43e441794013e37feb97b9/components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt#L677) this [here you can see an example](https://github.com/mozilla-mobile/android-components/blob/cdef87e847f4272fcb43e441794013e37feb97b9/components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt#L612)

Reply:
Added the update store but GH isn't picking up the changes